framework/helper/randomisation.cpp

```diff
@@ -0,0 +1,32 @@
+#include "randomisation.h"
+#include "helper/logging.h"
+#include <time.h>
+#include <stdlib.h>
+#include <math.h>
+
+void T3_Init_Random(){
+    return srand(time(NULL));
+}
```

Comment:
it's void but returns something? that is not a good idea. 

Also, this srand is what we use for randomization seed and I think it usually needs to be done only once (not sure though. needs testing) If that's the case, maybe we should just srand(time(NULL)); somewhere on our T3_Init method in main.cpp, and use no function for this one.

Reply:
True that, I removed the T3_Init_Random() function.

srand is used to initialize the randomisation seed (using the time(NULL) as a reference, so it takes the seed from the current time). Every time the srand function is called a new "pseudo random" seed should be initialized, so it should not be called more than one time (reference: [c - srand() - StackOverflow](https://stackoverflow.com/questions/7343833/srand-why-call-it-only-once#:~:text=To%20avoid%20this%20problem%2C%20srand,different%20sequence%20of%20random%20numbers.)

I added the srand(time(NULL)) function in the main.cpp file, inside the T3_Init() method, so the seed gets randomised once the program is started.